doc/conf.py

```diff
@@ -206,5 +206,5 @@
 # Enable nitpicky mode - which ensures that all references in the docs
 # resolve.
 
-nitpicky = True
-nitpick_ignore = []
+nitpicky = False
+nitpick_ignore = [('py:class', 'type')]
```

Comment:
It would be better to make the second part more specific, it's not clear where this comes from. See for example what we have in MNE where it's more specific (and thus clearer why it's there, and less likely to catch something that shouldn't be caught):

https://github.com/mne-tools/mne-python/blob/0ba06ba6b4dc446146d9b1a3764a697cd4ab16d2/doc/conf.py#L472-L483

Reply:
Added these back in. I think I wasn't sure how to get things like "`SpectralConnectivity`" to actually find the object. I was doing something like

:class:`SpectralConnectivity`.

Reply:
even
```
:class:`SpectralConnectivity`
```
will work so long as whatever is being documented at the time is documented in the API docs at the same level as `SpectralConnectivity` (e.g., if you're putting this ref in `mne_connectivity.phase_slope_index` function and the class is documented as `mne_connectivity.SpectralConnectivity` and hence at the same `mne_connectivity` level). If you remove this and have problems let me know and I can probably help find a solution that links properly